client/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "rootDir": "src",
     "rootDirs": [
       ".",
-      "../dist"
+      "../dist",
+      "../bazel-bin"
```

Comment:
for editor support I'm guessing? or is this load bearing for debugging as well?

Reply:
I'll revert this. I was seeing an error in my editor that was saying something along the lines of `...dist/common/resolver` wasn't built from source `...common/resolver.ts`. Which could make sense because we don't actually build anything with `tsc` and put it in the `dist` folder.